core-bundle/src/Command/DebugPagesCommand.php

```diff
@@ -86,13 +86,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
             $rows[] = [
                 $type,
-                $config ? $config->getPath() : '*',
-                $config ? $config->getUrlSuffix() : '*',
+                $config ? $config->getPath() ?: '-' : '*',
+                $config ? $config->getUrlSuffix() ?: '-' : '*',
                 $this->pageRegistry->supportsContentComposition($page) ? 'yes' : 'no',
                 isset($this->routeEnhancers[$type]) ? \get_class($this->routeEnhancers[$type]) : '-',
-                $config ? $this->generateArray($config->getRequirements()) : '-',
-                $config ? $this->generateArray($config->getDefaults()) : '-',
-                $config ? $this->generateArray($config->getOptions()) : '-',
+                $config ? $this->generateArray($config->getRequirements()) ?: '-' : '-',
```

Comment:
we should return `-` in `generateArray` of the array is empty instead of handling it here.

Reply:
Yes, I like this.